docs/docs/how-tos/nebari-idle-culler.md

```diff
@@ -0,0 +1,58 @@
+---
+id: idle-culling
+title: Updating JupyterHub Idle-Culling Settings
+description: How to update JupyterHub idle-culling settings
+---
+
+This document provides information on how to update JupyterHub idle-culling settings.
+
+## Overview
+
+The idle culler automatically shuts down user notebook servers after a certain period of inactivity to reduce the total resource usage from your user instances. There are two ways to cull idle notebooks and kernels:
+
+- Setting configuration options for the notebook server to shutdown kernels and the server itself after a certain period of inactivity.
+  The default kernel manager, the MappingKernelManager, can be configured to cull idle kernels. Its configuration is documented in more details at [NotebookApp](https://jupyter-notebook.readthedocs.io/en/stable/config.html#options).
+- Using an external JupyterHub culler ([jupyterhub-idle-culler](https://github.com/jupyterhub/jupyterhub-idle-culler)) to shutdown the whole server after a certain period of inactivity.
+  The external culler is more coarse-grained, while the internal culler is considered more granular and can be smarter. It is recommended to use both methods to avoid false-positives.
```

Comment:
We share the same message in the very next paragraph, so maybe we can remove it from here?